src/components/layout/header.tsx

```diff
@@ -1,12 +1,15 @@
 'use client';
 
-// import { useState } from 'react';
+import Link from 'next/link';
+import { useAtom } from 'jotai';
 
 import Gnb from '~/src/components/common/gnb';
+import ProfileDropdown from '~/src/components/common/profile-dropdown';
+import { accessTokenAtom } from '~/src/store/auth-store';
 import { cn } from '~/src/utils/class-name';
 
 export default function Header() {
-  // const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [accessToken] = useAtom(accessTokenAtom);
```

Comment:
```suggestion
  const accessToken = useAtomValue(accessTokenAtom);
```

useAtomValue라고 value만 가져올수 있는 훅도 존재합니다! 이것도 필수 수정사항은 아니고 참고사항입니다~!